docs/commands/super-db.md

```diff
@@ -290,21 +296,21 @@ optimize scans over such data is impaired.
 
 Because commits are transactional and immutable, a query
 sees its entire data scan as a fixed "snapshot" with respect to the
-commit history.  In fact, Zed's [from operator](../language/operators/from.md)
+commit history.  In fact, the [`from` operator](../language/operators/from.md)
 allows a commit object to be specified with the `@` suffix to a
 pool reference, e.g.,
 ```
-zed query 'from logs@1tRxi7zjT7oKxCBwwZ0rbaiLRxb |> ...'
+super db query 'from logs@1tRxi7zjT7oKxCBwwZ0rbaiLRxb | ...'
 ```
 In this way, a query can time-travel through the commit history.  As long as the
-underlying data has not been deleted, arbitrarily old snapshots of the Zed
+underlying data has not been deleted, arbitrarily old snapshots of the
 lake can be easily queried.
 
-If a writer commits data after and while a reader is scanning, then the reader
+If a writer commits data after or while a reader is scanning, then the reader
```

Comment:
Wanted to call extra attention to this change. The `and` didn't read right. But if I'm missing something, please correct me.